app/assets/javascripts/invoice.js

```diff
@@ -56,8 +56,9 @@ $(document).on('change',".selectclient", function() {
 
         },
         success: function (data) {
-            var str = data.street_1+" "+data.street_2+", <br/>"+data.city+", "+data.state+", <br/>"+data.pincode.to_s
-            $("#client_address").html(str);
+            var state_string=  data.state
```

Comment:
space between variable and `=` required. Also follow JS variable naming convention https://github.com/airbnb/javascript#22.2